crates/autopilot/src/infra/persistence/dto/fee_policy.rs

```diff
@@ -25,44 +27,59 @@ impl FeePolicy {
                 order_uid: boundary::database::byte_array::ByteArray(order_uid.0),
                 kind: FeePolicyKind::Surplus,
                 surplus_factor: Some(factor),
-                max_volume_factor: Some(max_volume_factor),
+                surplus_max_volume_factor: Some(max_volume_factor),
                 volume_factor: None,
+                price_improvement_factor: None,
+                price_improvement_max_volume_factor: None,
             },
             domain::fee::Policy::Volume { factor } => Self {
                 auction_id,
                 order_uid: boundary::database::byte_array::ByteArray(order_uid.0),
                 kind: FeePolicyKind::Volume,
                 surplus_factor: None,
-                max_volume_factor: None,
+                surplus_max_volume_factor: None,
                 volume_factor: Some(factor),
+                price_improvement_factor: None,
+                price_improvement_max_volume_factor: None,
             },
             domain::fee::Policy::PriceImprovement {
                 factor,
                 max_volume_factor,
-                ..
+                quote: _,
             } => Self {
                 auction_id,
                 order_uid: boundary::database::byte_array::ByteArray(order_uid.0),
                 kind: FeePolicyKind::PriceImprovement,
-                surplus_factor: Some(factor),
-                max_volume_factor: Some(max_volume_factor),
+                surplus_factor: None,
+                surplus_max_volume_factor: None,
                 volume_factor: None,
+                price_improvement_factor: Some(factor),
+                price_improvement_max_volume_factor: Some(max_volume_factor),
             },
         }
     }
-}
 
-impl From<FeePolicy> for domain::fee::Policy {
-    fn from(row: FeePolicy) -> domain::fee::Policy {
-        match row.kind {
+    #[allow(dead_code)]
```

Comment:
Why is this dead code now? Can we remove?

Reply:
It was never used. Removed.